portfolio/src/main/webapp/script.js

```diff
@@ -237,10 +237,13 @@ function appendTextToList(comment, ulElement) {
 
   liElement.appendChild(infoDivElement);
   const textPElement = appendPTagToContainer(comment.text, liElement);
+  textPElement.className = COMMENT_CLASS;
   liElement.appendChild(createCommentImage(comment.imageUrl));
+
+  const sentimentPElement = appendPTagToContainer(comment.sentiment, liElement);
```

Comment:
I see that the sentiment score has a lot of decimal places. I think it would be more intuitive to reduce that down to maybe 2 decimal places. Also how would the user interprete the score on your UI? Perhaps you want a better print out like smiley face or sad face or so?

Reply:
Ah nvm, I see you're doing  this in a subsequent pull request.

Reply:
Ah sorry, I should have mentioned that the layout was just temporary. I updated my description to state this.